src/odb/include/odb/dbBlockCallBackObj.h

```diff
@@ -85,6 +85,7 @@ class dbBlockCallBackObj
   // dbNet Start
   virtual void inDbNetCreate(dbNet*) {}
   virtual void inDbNetDestroy(dbNet*) {}
+  virtual void inDbNetMerge(dbNet*, dbNet*) {}
```

Comment:
The should be inDbNetPreMerge or inDbNetPostMerge depending on where the callback happens relative to the edits (or both if needed).